src/client/src/containers/PrivacyModal/index.tsx

```diff
@@ -0,0 +1,124 @@
+import classnames from "classnames";
+import * as React from "react";
+import { connect } from "react-redux";
+
+import asModal from "../../components/Modal";
+import buttonStyles from "../../css/buttonStyles.module.css";
+import styles from "./styles.module.css";
+import { ReactComponent as Cancel } from "../../assets/cancel.svg";
+import { isPrivacyModalOpenSelector } from "../../selectors/modalSelector";
+
+import { defineMessages, injectIntl, InjectedIntlProps } from "react-intl";
+import { closeModalAction } from "../../actions/modalActions/modalActions";
+import { AppState } from "../../reducers";
+import { Dispatch } from "redux";
+import RootAction from "../../actions/ActionType";
+
+import { IDependency } from "../DependencyInfo";
+
+interface IStateProps {
+  isModalOpen: boolean;
+  dependency: IDependency;
+}
+
+interface IDispatchProps {
+  closeModal: () => any;
+}
+
+type Props = IStateProps & IDispatchProps & InjectedIntlProps;
+
+const messages = defineMessages({
+  beingRedirected: {
+    id: "privacyModal.beingRedirected",
+    defaultMessage: "You are being redirected."
+  },
+  thirdPartyWebsite: {
+    id: "privacyModal.thirdPartyWebsite",
+    defaultMessage:
+      "You will be taken to {thirdPartyURL} which is a non-Microsoft service."
+  },
+  privacyStatement: {
+    id: "privacyModal.privacyStatement",
+    defaultMessage: "Privacy Statement"
+  },
+  OK: {
+    id: "privacyModal.OK",
+    defaultMessage: "OK"
+  }
+});
+
+const PrivacyModal = (props: Props) => {
+  const { dependency, intl } = props;
+
+  if (dependency === null || dependency === undefined) {
+    return null;
+  }
+
+  const { downloadLink, privacyStatementLink } = dependency;
+
+  const cancelKeyDownHandler = (event: any) => {
+    if (event.keyCode === 13 || event.keyCode === 32) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode If you read this it states the keyCode api is deprecated. Try using https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key instead, as its not deprecated and it's more readable. (More docs: https://www.w3schools.com/jsref/event_key_key.asp)

Also you should change the type of the event from any to a KeyboardEvent.

Reply:
Hey, I think it would be better to address this with a separate PR because we are using keyCode api for our other key handlers. I updated the type of event though

Reply:
Yeah you're right. Hopefully this will be addressed soon.